appveyor.yml

```diff
@@ -27,20 +27,21 @@ environment:
 
 install:
   - powershell -file .github\appveyor-prepare-mssql.ps1
-  - set PATH=%PATH%;C:\msys64\mingw64\bin;%GOPATH%\bin
+  - set PATH=C:\msys64\usr\bin;C:\msys64\mingw64\bin;%GOPATH%\bin;%PATH%
+  - pacman -Rsc --noconfirm gcc
```

Comment:
Why?

Reply:
MSYS2 has 2 versions of `make` and `gcc` which are preinstalled in AppVeyor images:

1. `pacman -S make gcc`
2. `pacman -S mingw-w64-x86_64-make mingw-w64-x86_64-gcc`

SQLite driver requires MinGW version:

- [sqlite3_windows.go](https://github.com/mattn/go-sqlite3/blob/3fb7a0e792edd47bf0cf1e919dfc14e2be412e15/sqlite3_windows.go#L12)
- https://github.com/mattn/go-sqlite3/issues/392#issuecomment-287047970

But MinGW's `make` has inconvenient name.

My trick is to use the "ordinary" `make` and MinGW's `gcc`. The same effect can probably be achieved by reordering dirs in `PATH` but I have chosen to uninstall it. Might not be the best option.

Also I moved MSYS paths to the beginning so that other Cygwin utilities (from Git for example) don't interfere.

Reply:
Cool! May you please add SQLite3 to the AppVeyor test matrix, so we don't accidentally break it?